sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/text/TextTable.scala

```diff
@@ -28,11 +28,11 @@ import org.apache.spark.sql.util.CaseInsensitiveStringMap
 case class TextTable(
     name: String,
     sparkSession: SparkSession,
-    options: CaseInsensitiveStringMap,
+    originalOptions: CaseInsensitiveStringMap,
```

Comment:
do we still need to rename it?

Reply:
We can revert this back like other `FileTable`.

Reply:
Fixed.